include/Core/Terms/Modules/DeleteTermsByPostCountModule.php

```diff
@@ -35,17 +35,17 @@ public function render() {
 
 			<?php _e( 'Delete Terms if the post count is ', 'bulk-delete' ); ?>
 			<?php $this->render_number_comparison_operators(); ?>
-			<input type="number" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_post_count" placeholder="<?php _e( 'Post count', 'bulk-delete' ); ?>">
+			<input type="number" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>" placeholder="<?php _e( 'Post count', 'bulk-delete' ); ?>" min="0">
 		</fieldset>
 
 		<?php
 		$this->render_submit_button();
 	}
 
 	public function filter_js_array( $js_array ) {
-		$js_array['validators'][ $this->action ] = 'validatePostTypeSelect2';
-		$js_array['error_msg'][ $this->action ]  = 'selectPostType';
-		$js_array['msg']['selectPostType']       = __( 'Please select at least one post type', 'bulk-delete' );
+		$js_array['validators'][ $this->action ] = 'validatePostCount';
+		$js_array['error_msg'][ $this->action ]  = 'validPostCount';
+		$js_array['msg']['validPostCount']       = __( 'Please enter valid post count(Greater than or Equal to zero)', 'bulk-delete' );
```

Comment:
Let's change this message to `Please enter the post count based on which terms should be deleted. A valid post count will be greater than or equal to zero`

Reply:
Fixed.